coordinatorlib/src/it/scala/com/socrata/datacoordinator/truth/metadata/sql/PostgresDatasetMapWriterTest.scala

```diff
@@ -29,12 +55,14 @@ class PostgresDatasetMapWriterTest extends FunSuite with MustMatchers with Befor
   }
   val ZeroID = 0L
 
+  val resourcName = Some("1234-abcc")
```

Comment:
Although it is too late and not going to happen.  Could we start a name or identifier with an alphabet and not a number?